### PR TITLE
[new release] bistro and bistro-bio (0.6.0)

### PR DESCRIPTION
**Pull request by pveber**
A library to build and run distributed scientific workflows

- Project page: <a href="https://github.com/pveber/bistro/">https://github.com/pveber/bistro/</a>

##### CHANGES:

core library:
- added ifelse and trywith constructs
- moved bio-stuff in new package bistro-bio

engine:
- precious items
- repo clipping

ppx:
- include script mechanism

bio:
- added Bedtools.merge
- added Samtools.faidx
- added SE_or_PE
- new wrapper rnaspades
- new wrapper busco

### CODE REVIEW

**Comment by pveber** [comment]
The remaining compilation failure is due to a missing constraint in `future` I believe.

File: packages/bistro-bio/bistro-bio.0.6.0/opam [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+bug-reports: "https://github.com/pveber/bistro/issues"
+depends: [
+  "dune" {>= "1.11"}
+  "biocaml"
+  "biotk"
+  "tyxml"
+  "bistro"
+]
+build: [
+  ["dune" "subst"] {pinned}
```
Comment (mseri) [inline packages/bistro-bio/bistro-bio.0.6.0/opam:22]:
```suggestion
  ["dune" "subst"] {dev}
```

File: packages/bistro/bistro.0.6.0/opam [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+"""
+maintainer: ["philippe.veber@gmail.com"]
+authors: ["Philippe Veber"]
+license: "CeCILL-B"
+homepage: "https://github.com/pveber/bistro/"
+bug-reports: "https://github.com/pveber/bistro/issues"
+depends: [
+  "base64" {>= "3.0.0"}
+  "bos"
+  "dune" {> "1.6"}
```
Comment (mseri) [inline packages/bistro/bistro.0.6.0/opam:28]:
```suggestion
  "dune" {> "1.11"}
```

File: packages/bistro/bistro.0.6.0/opam [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+  "ocamlgraph" {>= "1.8.7"}
+  "ocaml" {>= "4.08.0"}
+  "ppxlib" {>= "0.23.0"}
+  "ppx_sexp_conv"
+  "rresult"
+  "sexplib" {>= "113.24.00"}
+  "tyxml" {>= "4.3.0"}
+]
+build: [
+  ["dune" "subst"] {pinned}
```
Comment (mseri) [inline packages/bistro/bistro.0.6.0/opam:41]:
```suggestion
  ["dune" "subst"] {dev}
```

**Comment by mseri** [comment]
> The remaining compilation failure is due to a missing constraint in `future` I believe.

Indeed, seems that biocaml needs to have a lower bound on core_kernel, although I am not sure which yet

**Comment by pveber** [comment]
> Indeed, seems that biocaml needs to have a lower bound on core_kernel, although I am not sure which yet

I'll investigate and submit a separate PR.

**Comment by mseri** [comment]
Thanks

**Pull request merged**